### DISCUSSION TITLE
MAUI WinUI integration instead of separate "reference Project"

### DISCUSSION THREAD

**Discussion by Zumpel96** [dotnet/maui#1919] [Ideas]
I just finished the `Getting started` part of MAUI and I can execute the Android platform without any problems.
Anyways I stumbled across something, which looks, in my eyes, a little bit unnecessary.

When you create a new MAUI project (for example `HelloMaui`) and generate a new solution, you get two projects.
One for WinUI `HelloMaui.WinUI` and one for all the "other" platforms `HelloMaui`  (Android, iOS, MacCatalyst). 

However, why is there another project?

Within the WinUI project, you only have references to the "main" project and therefore, in a plain vanilla environment, just a 1:1 project, which only consists of references, without any differences.

I really do not understand, why this is done, since another subdirectory in `Platforms` called `WinUI` or simply just `Windows` would definitely be clear enough. I didn't mess around with VS2021 yet, but from experience I am trying to minimize the number of concurrently loaded projects to the performance issues of previous VS versions. It would also make the solution a lot more structured.

**Comment by BreyerW** [comment]
This is preview artifact and should be gone in next preview if everything goes well. If not it will be gone in later previews or release candidates
